book/src/external-attacher.md

```diff
@@ -6,8 +6,9 @@
 
 **Status:** GA/Stable
 
-Latests stable release | Branch | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
+Latest stable release | Branch | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
```

Comment:
Discussed offline.

*Let's rename `Compatible with CSI Version` to `Min CSI Version`.
* And add a column for `Max CSI Version`

Max should be empty for everything except the v0.x.x releases.